.travis.rosinstall.melodic

```diff
@@ -0,0 +1,11 @@
+# hrpsys and hrpsys_ros_bridge are not released in melodic
+- git:
+    uri: https://github.com/fkanehiro/hrpsys-base
+    local-name: hrpsys
+- git:
+    uri: https://github.com/start-jsk/rtmros_common
+    local-name: rtm-ros-robotics/rtmros_common
```

Comment:
We need to wait for https://github.com/ros/rosdistro/pull/24678 to be merged and applied in order to remove hrpsys and rtmros_common from this file.
Once hrpsys 315.15.0-8 appears on http://repositories.ros.org/status_page/melodic_default.html?q=hrpsys, we can remove them because we use `ros-testing` for ROS repo:
https://github.com/jsk-ros-pkg/jsk_travis/blob/0.5.7/travis.sh#L183

Reply:
hrpsys 315.15.0-8 seems released to `ros-testing`.
@Naoki-Hiraoka Could you try removing hrpsys and rtmros_common from this file?

Reply:
Thank you. I fixed.

Reply:
I think we need to keep hrpsys and rtmros_common in .rosinstall.melodic .
https://github.com/start-jsk/rtmros_tutorials/pull/571#issuecomment-627673303

Reply:
I understand. Thank you.